src/app/components/mno-select-app-wizard/mno-select-app-wizard.html

```diff
@@ -0,0 +1,45 @@
+<div id="select-app">
+  <div class="select-app-marketplace">
+    <div class="row">
+      <div class="col-xs-12">
+
+        <mno-widget class="top-buffer-2">
```

Comment:
Are you sur the library `mno-ui-elements` is properly loaded? In the screenshot I don't see what a widget looks like in the lib.